packages/translations/src/languages/nb.ts

```diff
@@ -449,6 +449,7 @@ export const nbTranslations: DefaultTranslationsObject = {
     previouslyPublished: 'Tidligere Publisert',
     problemRestoringVersion: 'Det oppstod et problem med gjenoppretting av denne versjonen',
     publish: 'Publisere',
+    publishAllLocales: 'Publiser alle lokaliteter',
```

Comment:
```suggestion
    publishAllLocales: 'Publiser alle språk',
```

Great PR. As a native Norwegian speaker, I’d suggest changing this. The term ‘lokaliteter’ sounds less natural in this context, and ‘språk’ is a more standard way to refer to languages.

Reply:
Thank you @andershermansen 🙏